functorch/experimental/control_flow.py

```diff
@@ -1,8 +1,10 @@
-from torch import cond  # noqa: F401
+# TODO: Currently cond is not fully exposed under torch, e.g. torch.cond, but it causes import errors 
```

Comment:
oh, this might be because of an circular import. We can break tie in cond.py by moving 
```
from torch._functorch.aot_autograd import AOTConfig, create_joint, from_fun
``` 
into `create_fw_bw_graph` and leave these imports unmodified.

Reply:
I have moved the import into the `create_fw_bw_graph` as you suggested and this resolved the circular import issue